gen/gen.go

```diff
@@ -93,11 +101,29 @@ func (g *Gen) Build(config *Config) error {
 		}
 	}
 
+	var overrides map[string]string
+	if config.OverridesFile != "" {
```

Comment:
what if use `os.Stat` in order to detect there is the override file (.swagoveride) in the working directory and we try to load and parse instead of waiting for a command line param?

Reply:
It already does that. Right now it looks for .swaggo by default and only does something else if the flag is provided, but can easily change that to be .swagoverride.

Reply:
`.swaggo` is fine